subworkflows/local/kofamscan.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run KOFAMSCAN on protein fasta from orf_caller output
+//
+
+include { KOFAMSCAN     } from '../../modules/local/eukulele/kofamscan'
+include { DOWNLOAD_KODB } from '../../modules/local/eukulele/download_kofamscan_db'
+
+workflow SUB_KOFAMSCAN {
```

Comment:
This would be nice as an nf-core subworkflow I think. Whether you think that's well spent time or not is up to you.
In any case, the naming principles for subworkflows are here: https://nf-co.re/docs/contributing/subworkflows#naming-conventions.